tools/data_integrity/fix_candidate_age.php

```diff
@@ -84,11 +84,16 @@
             JOIN candidate c ON c.CandID=s.CandID
             JOIN test_names tn ON tn.ID=f.TestID
         WHERE c.Active='Y' AND s.Active='Y'
-
-        AND tn.Test_name=:tn",
+            AND f.DataID IS NOT NULL
```

Comment:
What about instruments that use tables instead of json data?

Reply:
I was under the impression that instruments that use tables _also_ populate the instrument_data table. Was that behaviour changed between CCNA's v24 and LORIS now?

Reply:
Dave thinks @skarya22 is correct